pyvista/core/pointset.py

```diff
@@ -1280,11 +1280,18 @@ def plot_curvature(self, curv_type='mean', **kwargs):
     def volume(self) -> float:
         """Compute the volume of the point grid.
 
-        This extracts the external surface and computes the interior
-        volume.
+        .. versionchanged:: 0.37.0
+           This now calculates the sum of the volume of all cells.
+
         """
-        surf = self.extract_surface().triangulate()
-        return surf.volume
+        vols = self.compute_cell_sizes(length=False, area=False, volume=True)
+        return np.sum(vols["Volume"])
+
+    @property
+    def area(self) -> float:
+        """Compute the area of the point grid if 2D."""
+        vols = self.compute_cell_sizes(length=False, area=True, volume=False)
+        return np.sum(vols["Area"])
```

Comment:
We're talking about microseconds here overall, but invoking the method on the ndarray subclass seems to be 30% faster (on a 2d grid with 10k cells):
```suggestion
        return vols["Area"].sum()
```
(And the same for `volume()`).